lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -18,6 +18,11 @@ module Cardano.Pool.Metrics
     , combineMetrics
     , ErrMetricsInconsistency (..)
 
+      -- * StakePoolLayer
+    , StakePoolLayer (..)
```

Comment:
Maybe this should be a different module, but sure

Reply:
If there's no particular reason to have it in a separate module, then let's not for now.